sdk/onlineexperimentation/arm-onlineexperimentation/review/arm-onlineexperimentation.api.md

```diff
@@ -147,26 +147,26 @@ export interface OnlineExperimentationClientOptionalParams extends ClientOptions
 }
 
 // @public
-export interface OnlineExperimentationWorkspaceSku {
-    name: OnlineExperimentationWorkspaceSkuName;
-    readonly tier?: OnlineExperimentationWorkspaceSkuTier;
+export interface OnlineExperimentationWorkspace extends TrackedResource {
+    identity?: ManagedServiceIdentity;
+    properties?: OnlineExperimentationWorkspaceProperties;
+    sku?: OnlineExperimentationWorkspaceSku;
 }
 
 // @public
-export type OnlineExperimentationWorkspaceSkuName = string;
-
-// @public
-export type OnlineExperimentationWorkspaceSkuTier = string;
-
-// @public
-export interface OnlineExperimentWorkspace extends TrackedResource {
+export interface OnlineExperimentationWorkspacePatch {
     identity?: ManagedServiceIdentity;
-    properties?: OnlineExperimentWorkspaceProperties;
+    properties?: {
```

Comment:
it seems the anomymous model, could you share the spec with me?

Reply:
https://github.com/openapi-env-test/azure-rest-api-specs/commit/2b341fca86a5d4ff863d88084e358cacd7b4e904#diff-d09788d3ab3ac94375e56663150a61ddb951c3d42055885cb1ad8776531d0f60

Reply:
Accept this since this is preview version.